packages/api/src/command/medical/document/document-query.ts

```diff
@@ -70,6 +70,10 @@ export async function queryDocumentsAcrossHIEs({
 
   const patient = await getPatientOrFail({ id: patientId, cxId });
 
+  if (patient.hieOptOut) {
+    throw new BadRequestError("Patient has opted out of pulling from and sharing to the hies");
```

Comment:
nit: all caps on HIEs (or should we use "networks"?)

Reply:
"Patient has opted out from the networks"